_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
-msgstr ""
+msgstr "Beachte dass das Abhören des Server Signals zwar sicherstellt, dass du mit den anderen Musikern synchron bist, dass es aber auch zu Problemen kommen kann, wenn die Gesamtlatenz (angezeigt durch die \"Delay\"-Lampe in Jamulus) die meiste Zeit über nicht grün oder zumindest gelb ist. Lies das [Software-Handbuch](/wiki/Software-Manual), um zu erfahren, wie du dein Setup in diesem Fall anpassen kannst."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:52
 msgid "### Can't work out your mic settings?"
-msgstr ""
+msgstr "### Du kannst deine Mikrofoneinstellungen nicht herausfinden?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:54
 msgid "When using a microphone while playing your other instrument, you can use a stereo audio input signal in your settings where one channel is connected to the instrument and the other channel is connected to a microphone signal. On the microphone channel an optional reverberation effect can be applied."
-msgstr ""
+msgstr "Wenn du ein Mikrofon verwendest, während du ein anderes Instrument spielst, kannst du in deinen Einstellungen ein Stereo-Audioeingangssignal verwenden, bei dem ein Kanal mit dem Instrument und der andere Kanal mit einem Mikrofonsignal verbunden ist. Auf den Mikrofonkanal kann ein optionaler Hall-Effekt eingestellt werden."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:56
 msgid "### Buffer LEDs suddenly going red, outages, jittering, weird sounds?"
-msgstr ""
+msgstr "### Puffer-LEDs leuchten plötzlich rot, Ausfälle, [Jitter](https://de.wikipedia.org/wiki/Jitter), seltsame Geräusche?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:58
 msgid "The CPU in your computer may be having issues. Try not to have anything competing with Jamulus (like Zoom Meetings or Facebook live streams) on your machine. Or at least quit them while you're playing. Prevent things like virus scanners doing scans, or software updates happening, etc. Be aware that the reverb setting in Jamulus also uses more CPU the more reverb you have."
-msgstr ""
+msgstr "Möglicherweise gibt es Probleme mit der CPU deines Computers. Versuche alles was mit Jamulus konkurriert (wie Zoom Meetings oder Facebook-Livestreams) von deinem Computer zu entfernen. Oder beende diese zumindest, während du spielst. Verhindere dass z. B. Virenscanner Scans durchführen oder Software-Updates durchgeführt werden, usw. Beachte dass auch die Hall-Einstellung in Jamulus mehr CPU verbraucht, je mehr Hall eingestellt wird."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:60
 msgid "### Ping times and latency start well, then get worse, causing issues"
-msgstr ""
+msgstr "### Ping-Zeiten und Latenzzeiten beginnen gut, werden dann schlechter und verursachen Probleme"
```

Comment:
Not part of this PR, but 
Ping-Zeiten und Latenzzeiten --> Latenzzeiten

I'd rather remove Ping

@gilgongo

Reply:
Can you explain? "Ping" is a label the UI, so it's a bit unwise to remove it here or it won't be clear what's being referred to.